visualizer.py

```diff
@@ -0,0 +1,39 @@
+from termcolor import colored
+
+
+class Bar_chart:
+    def __init__(self, monthly_data):
+        self.monthly_data = monthly_data
```

Comment:
`monthly_data` does not have the information for the day. Are we assigning days here on bases of the data's order in the list?

Reply:
it contains all the days in order so i dont need to parse the PKT column to find the day and then print it . i can iterate through it